_sass/setup/_font-face.scss

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap');
```

Comment:
I'm fine with utilizing Google's font api, but I'd prefer that we're consistent.

Reply:
Sorry, I had to talk to you about fonts.
1. Poppins is base font. (we can load Google font) and in Bootstrap variable change default value with Poppins.
And in the following way we will be able to delete some extra css code.
2. In Font-face what was add I din not find thin font-weight (and I add this Google font) didn't found poppins-thin-webfornt